src/components/base/accordion/Accordion.js

```diff
@@ -151,23 +186,26 @@ export default function Accordion({
   data = null,
   className = "",
   children,
+  isLoading,
 }) {
   useEffect(() => {
     firstAccordionRender = false;
   }, []);
 
-  data =
-    data &&
-    data.map((a, i) => (
-      <Item
-        title={a.title}
-        subTitle={a.subTitle}
-        key={`${a.title}_${i}`}
-        eventKey={a.key || i.toString()}
-      >
-        <BodyParser body={a.content} />
-      </Item>
-    ));
+  if (isLoading) {
+    return <AccordionSkeleton className={className} />;
+  }
```

Comment:
Har ikke noget med react-bootstrap opdateringen at gøre - men fixer en grim skeleton fejl på hjælpesiden (skeleton dækker for alt indhold)